### ISSUE TITLE
Conflicting dependencies in sentry-sdk and sentry-sdk[flask]

### ISSUE THREAD

**Issue by chtnnh** [rafaeelaudibert/Encryptor#33]
When running `pip install -r requirements.txt` as instructed in the `README.md`, the following error causes the execution to stop:

```
INFO: pip is looking at multiple versions of sentry-sdk to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of python-dotenv to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of pytest-mock to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of pytest-flask to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of pytest to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of markupsafe to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of itsdangerous to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of flask to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of contextlib2 to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of click to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of blowfish to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of blinker to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install sentry-sdk==0.19.0 and sentry-sdk[flask]==0.12.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested sentry-sdk==0.19.0
    sentry-sdk[flask] 0.12.2 depends on sentry-sdk 0.12.2 (from https://files.pythonhosted.org/packages/cf/c2/845d503adb168fcd276a54ac7e5a4fab18d8ed83cf8ad8fba6599eb6ca9c/sentry_sdk-0.12.2-py2.py3-none-any.whl#sha256=ab16b539c40cc18e3dcfec99d0d3fff16337a050eaf79d4503ef2b73af9a7d9e (from https://pypi.org/simple/sentry-sdk/))

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

As pip suggests, these can be fixed by either loosening the sentry sdk version requirements or removing the package versions altogether.

**Comment by rafaeelaudibert** [comment]
We'll probably need `sentry-sdk[flask]` for it to properly integrate with `flask`, so I'd rather you not remove it.

**Comment by chtnnh** [comment]
> We'll probably need `sentry-sdk[flask]` for it to properly integrate with `flask`, so I'd rather you not remove it.

Oh, I didn't remove sentry-sdk[flask], I just removed the version number so it automatically picks up the latest one. The event is showing up in sentry so I am sure it is working as expected. Let me know if that is acceptable and I will file a PR asap

**Comment by rafaeelaudibert** [comment]
Oh, that makes sense! Of course, go ahead and open a PR :)

**Comment by rafaeelaudibert** [comment]
Closed by #35
